kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/SectionOrder.vue

```diff
@@ -314,6 +314,23 @@
         );
         return displaySectionTitle(section, sectionIndexOrder).toUpperCase();
       },
+
+      moveUplabel() {
+        return this.$tr('up');
+      },
+      moveDownLabel() {
+        return this.$tr('down');
+      },
```

Comment:
This appears to be a common pattern where you're wrapping the string in a computed property but you could just put `$tr('up')` directly into where you're passing the value to the prop like so: `:moveUpText="$tr('up')"`

Reply:
I realize after writing this that the expected prop is a `function` - so perhaps you could save some trouble with: `:moveUpText="() => $tr('up')"` if you wanted to reduce the code added here.

Reply:
Also, these are stylistic notes and are not blocking - the code works great as-is and is perfectly readable, no need to change things around unless you're inclined to personally